### PR TITLE
docs: Community resources on the front page

### PR DESCRIPTION
**Pull request by sbose78**
Community resources in the https://github.com/argoproj/argoproj/blob/master/community/membership.md#community-membership and https://argo-cd.readthedocs.io/en/stable/SUPPORT/ weren't too discoverable and therefore, I propose we have more details on the readme.

---

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,21 @@ Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes.
...
+### Contribution, Discussion and Support
+
+You can reach the ArgoCD community and developers via the following channels:
+
+* Q & A : [Github Discussions](https://github.com/argoproj/argo-cd/discussions)
+* Chat : [The #argo-cd Slack channel](https://argoproj.github.io/community/join-slack)
+* Contributors Office Hours: [Every Thursday](https://calendar.google.com/calendar/u/0/embed?src=argoproj@gmail.com) | [Agenda](https://docs.google.com/document/d/1ttgw98MO45Dq7ZUHpIiOIEfbyeitKHNfMjbY5dLLMKQ)
+* User Community meeting: [Every other Wednesday](https://calendar.google.com/calendar/u/0/embed?src=argoproj@gmail.com) | [Agenda](https://docs.google.com/document/d/1xkoFkVviB70YBzSEa4bDnu-rUZ1sIFtwKKG1Uw8XsY8)
+
+Please go through the [membership Process](https://github.com/argoproj/argoproj/blob/master/community/membership.md#community-membership) if you would like to get involved in the ArgoCD project!
```
Comment (jannfis) [inline README.md]: I think we don't require a membership for getting involved :)

Comment (sbose78) [reply README.md]: Didn;t mean to make it sound like that, but you are right 🤕

Comment (sbose78) [reply README.md]: remove it altogether

File: README.md [hunk 2]
```diff
@@ -28,7 +28,22 @@ Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes.
...
+
+You can reach the ArgoCD community and developers via the following channels:
+
+* Q & A : [Github Discussions](https://github.com/argoproj/argo-cd/discussions)
+* Chat : [The #argo-cd Slack channel](https://argoproj.github.io/community/join-slack)
+* Contributors Office Hours: [Every Thursday](https://calendar.google.com/calendar/u/0/embed?src=argoproj@gmail.com) | [Agenda](https://docs.google.com/document/d/1ttgw98MO45Dq7ZUHpIiOIEfbyeitKHNfMjbY5dLLMKQ)
+* User Community meeting: [Every other Wednesday](https://calendar.google.com/calendar/u/0/embed?src=argoproj@gmail.com) | [Agenda](https://docs.google.com/document/d/1xkoFkVviB70YBzSEa4bDnu-rUZ1sIFtwKKG1Uw8XsY8)
+
+
+Participation in the ArgoCD project is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
```
Comment (sbose78) [inline README.md:43]: cc @edlee2121

**Review by jannfis — APPROVED** [review]
LGTM

**Review by alexmt — APPROVED** [review]

**Pull request merged**